subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/declaring_repositories.adoc

```diff
@@ -1,837 +1,346 @@
 [[declaring-repositories]]
-= Declaring repositories
+= Declare Repositories
 
-Gradle can resolve dependencies from one or many repositories based on Maven, Ivy or flat directory formats.
-Check out the <<#sec:repository-types,full reference on all types of repositories>> for more information.
+Gradle allows you to define **repositories**: sources for dependency downloads.
+During a build, Gradle searches repositories to resolve dependencies.
+You can specify repositories as locations in a file system, or as URLs.
+Gradle can resolve dependencies from one or many repositories.
 
-[[sec:declaring_public_repository]]
-== Declaring a publicly-available repository
-
-Organizations building software may want to leverage public binary repositories to download and consume open source dependencies.
-Popular public repositories include <<#sub:maven_central,Maven Central>> and the <<#sub:maven_google,Google Android>> repository.
-Gradle provides built-in shorthand notations for these widely-used repositories.
-
-.Declaring a repository with the help of shorthand notations
-image::dependency-management-shortcut-repositories.png[]
-
-Under the covers Gradle resolves dependencies from the respective URL of the public repository defined by the shorthand notation.
-All shorthand notations are available via the link:{groovyDslPath}/org.gradle.api.artifacts.dsl.RepositoryHandler.html[RepositoryHandler] API.
-Alternatively, you can <<#sec:declaring_custom_repository,spell out the URL of the repository>> for more fine-grained control.
-
-[[sub:maven_central]]
-=== Maven Central repository
-
-Maven Central is a popular repository hosting open source libraries for consumption by Java projects.
-
-To declare the link:https://repo.maven.apache.org/maven2/[Maven Central repository] for your build add this to your script:
-
-.Adding central Maven repository
+[TIP]
 ====
-include::sample[dir="snippets/artifacts/defineRepository/groovy",files="build.gradle[tags=maven-central]"]
-include::sample[dir="snippets/artifacts/defineRepository/kotlin",files="build.gradle.kts[tags=maven-central]"]
+Can't find the information you're looking for?
```

Comment:
Interesting. I can add a bullet point list to each of these. My only concern is length -- bulleted lists in particular take up a LOT of space in admonitions, and it really breaks up the page. It can also push content below the fold, especially on smaller screens. Given that concern, would you still like to add the bulleted list?

Reply:
Your right that would break it up too much.  Perhaps adding just a brief description like "for advanced content about repositories such as authentication, remote protocols and local configurations"